src/content/en/2019/performance.md

```diff
@@ -0,0 +1,266 @@
+
+#### FID by geo
+
+<figure>
+// Chart: Bar distribution of 07_04d
+<figcaption>Figure 24. Distribution of websites labelled as having fast, average, or slow FID, broken down by geo.</figcaption>
+</figure>
+
+In this breakdown of FID by geographic location, Korea is out in front of everyone else again. But the top geos have some new faces: the US, Australia, and Canada are next with 35-40% of websites having fast FID.
+
+As with the other geo-specific results, there are so many possible factors that could be contributing to the user experience. For example, perhaps wealthier geos are more privileged to be able to spend more money on better network infrastructure and its residents have more money to spend on desktops and/or high-end mobile phones.
+
+## Conclusion
+
+Quantifying how fast a web page loads is an imperfect science that can't be represented by a single metric. Conventional metrics like `onload` can miss the mark entirely by measuring irrelevant or imperceptible parts of the user experience. User-perceived metrics like FCP and FID more faithfully convey what users see and feel. Even still, neither metric can be looked at in isolation to draw conclusions about whether the overall page load experience was fast or slow. Only by looking at many metrics holistically can we start to understand the performance for an individual website and the state of the web.
+
+The data presented in this chapter showed that there is still a lot of work to do to meet the goals set for fast websites. Certain form factors, effective connection types, and geos do correlate with better user experiences, but we can't forget about the combinations of demographics with poor performance. In many cases, the web platform is used for business; making more money from improving conversion rates can be a huge motivator for speeding up a website. Ultimately, for all websites, performance is about delivering positive experiences to users in a way that doesn't impede, frustrate, or enrage them.
+
+As the web gets another year older and our ability to measure how users experience it improves incrementally, I'm looking forward to developers having access to metrics that capture more of the holistic experience. FCP is very early on the timeline of showing useful content to users and newer metrics like [Largest Contentful Paint](https://web.dev/largest-contentful-paint) (LCP) are emerging to improve our visibility into how page loads are perceived. Newer metrics from the [Layout Instability API](https://web.dev/layout-instability-api) are giving us a new glimpse into the frustration users experience beyond page load. Equipped with these new metrics, the web in 2020 will become even more transparent, better understood, and give developers an advantage to make more meaningful progress to improve performance and contribute to positive user experiences.
```

Comment:
The conclusion mentions "Newer metrics" twice, and then "new metrics", ending up a bit repetitive. I'm thinking using some synonym might be better. Maybe replacing "Newer metrics from the Layout Instability API are..." with "The recent Layout Instability API is...".

Reply:
I agree! This stuck out to me too while rereading it. I'll rephrase per your suggestion.